modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1504,6 +1509,82 @@ If `group_id` is not specified, it defaults to the currently selected Organizati
 // CLI DEFAULTS
 include::partial$cli-default-options.adoc[tag=CLIdefaultOptions]
 
+=== exchange asset resource upload
+
+----
+> exchange asset resource upload [options] <assetIdentifier> <filepath>
+----
+
+This command uploads a resource from `<filepath>` to an asset portal described in `<assetIdentifier>`.
+
+Argument `assetIdentifier` should be formatted as follows: `([group_id]/)<asset_id>/<version>`. +
+If `group_id` is not specified, it defaults to the currently selected Organization ID.
+
+Supported file extensions: `jpeg`, `jpg`, `jpe`, `gif`, `bmp`, `png`, `webp`, `ico`, `svg`, `tiff`, `tif`.
+
+The successful output command will be a markdown codesnippet.
+You can use it on any page of your `<assetIdentifier>` asset.
+
+// CLI DEFAULTS
+include::partial$cli-default-options.adoc[tag=CLIdefaultOptions]
+
+=== exchange asset resource list
+
+----
+> exchange asset resource list [options] <assetIdentifier>
+----
+
+This command lists published resources for `<assetIdentifier>` in the asset portal
+
+Argument `assetIdentifier` should be formatted as follows: `([group_id]/)<asset_id>/<version>`. +
+If `group_id` is not specified, it defaults to the currently selected Organization ID.
+
+[NOTE]
+This command only supports published resources.
+
+You can use `--draft` option to list draft resources in the asset portal.
```

Comment:
Do we have other _types_ (?) here besides "published" or "draft"? 
Because it's easier to read everything in the same note, and we should change the wording a little bit:

```suggestion
[NOTE]
This command lists published resources by default. +
You can use the `--draft` option to list draft resources from the asset portal.
```